requirements/reproducible-linux.txt

```diff
@@ -983,9 +983,9 @@ tzlocal==5.2 \
     # via
     #   aws-sam-cli (setup.py)
     #   dateparser
-urllib3==1.26.19 \
-    --hash=sha256:37a0344459b199fce0e80b0d3569837ec6b6937435c5244e7fd73fa6006830f3 \
-    --hash=sha256:3e3d753a8618b86d7de333b4223005f68720bcd6a7d2bcb9fbd2229ec7c1e429
+urllib3==2.2.3 \
```

Comment:
Should we run integ tests against this PR to make sure start-api tests don't break?

Reply:
Yup, let me kick that off. `botocore` (and by extension `boto3`) did have this pinned, not entirely sure why Linux was left out of the equation when Windows was updated, but this should be safe.

Reply:
Tests are all good, no tests that talk to the AWS API broke as a result of the upgrade.